pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -152,28 +153,30 @@ func lastMandatoryExists(flowcontrolClientSet flowcontrolclient.FlowcontrolV1alp
 	return false, nil
 }
 
+const thisFieldManager = "api-priority-and-fairness-config-producer"
```

Comment:
oh nifty-- how about "v1.18-defaults"?

Reply:
Is there a way to read the version from variables or something, rather than code a literal in the controller?

Reply:
I do not see an easy way to plumb this correctly.  I think I would prefer to leave it to a later PR.

Reply:
Seems fine as a constant at least for now.

Reply:
Yeah I was just recommending putting "-v1" in the constant, we can change it later if needed. We can also wait and do this for the next version, or never.